index.html

```diff
@@ -23,29 +23,227 @@
+            <li class="list-item">Back end dev</li>
+            <span class="list-item">•</span>
+            <li class="list-item">2015</li>
+          </ul>
+        </div>
+        <div class="work-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+        </div>
+        <div class="courses">
+          <ul class="course-list">
+            <li class="courses-list">html</li>
+            <li class="courses-list">css</li>
+            <li class="courses-list">JavaScript</li>
+          </ul>
+        </div>
+        <button>See Project</button>
+      </div>
+    </section>
+
+    <section class="portfolio-dashboard">
+      <div class="card-header">
+        <div class="course-header">
+          <img src="./Images/Snapshoot Portfolio-projects.png" alt="" class="image">
+        </div>
+      </div>
+      <div class="card-body">
+        <div class="title">Tonic</div>
+        <div class="links">
+          <ul class="link-items">
+            <li class="canopy">CANOPY</li>
+            <span class="list-item">•</span>
+            <li class="list-item">Back end dev</li>
+            <span class="list-item">•</span>
+            <li class="list-item">2015</li>
+          </ul>
+        </div>
+        <div class="work-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+        </div>
+        <div class="courses">
+          <ul class="course-list">
+            <li class="courses-list">html</li>
+            <li class="courses-list">css</li>
+            <li class="courses-list">JavaScript</li>
+          </ul>
+        </div>
+        <button>See Project</button>
+      </div>
+    </section>
+    <section class="portfolio-dashboard">
+        <div class="card-header">
+          <div class="header-image">
+            <img src="./Images/Professional-art-printing.png" alt="Dashboard" class="image">
+          </div>
+        </div>
+        <div class="card-body">
+          <div class="title">Multi-Post <br> Stories</div>
+          <div class="links">
+            <ul class="link-items">
+              <li class="canopy">CANOPY</li>
+              <span class="list-item">•</span>
+              <li class="list-item">Back end dev</li>
+              <span class="list-item">•</span>
+              <li class="list-item">2015</li>
+            </ul>
+          </div>
+          <div class="work-description">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+          </div>
+          <div class="courses">
+            <ul class="course-list">
+              <li class="courses-list">html</li>
+              <li class="courses-list">css</li>
+              <li class="courses-list">JavaScript</li>
+            </ul>
+          </div>
+          <button>See Project</button>
+        </div>
+    </section>
+      <section class="about-myself">

```

Comment:
Please note that its a requirement  that you use flexbox to align this sections properties. refer to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md):
<img width="1667" alt="Screenshot 2022-10-28 at 10 52 45" src="https://user-images.githubusercontent.com/48546992/198534366-ddff530f-56f2-40bc-8d5e-522c7c19e98b.png">